server/plugins/auth.ts

```diff
@@ -13,6 +13,7 @@ export type AuthHandler = (
 
 // The order defines priority: the first available plugin is used.
 // Always keep 'local' auth plugin at the end of the list; it should always be enabled.
+// @ts-ignore
```

Comment:
Same thing here, needs a comment as to why it's safe to ignore or a fix of the root cause